src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -311,11 +311,12 @@ class ComposerController(private val storage: KeyValueStorage,
             }
             is ComposerResult.UploadFile.Failure -> {
                 removeAttachmentByUUID(result.uuid)
-                scene.showAttachmentErrorDialog(result.filepath)
+                scene.showAttachmentErrorDialog(result.message)
                 handleNextUpload()
             }
             is ComposerResult.UploadFile.Unauthorized -> {
-                generalDataSource.submitRequest(GeneralRequest.DeviceRemoved(false))
+                scene.showAttachmentErrorDialog(result.message)
+                handleNextUpload()
```

Comment:
I think if it is unauthorized, the sequence should stop right there, otherwise you will see multiple error dialogs.

Reply:
What do you think about, removing all following attachments and just send one error dialog with message determined by the worker?

Reply:
There's no other choice because we don't have a retry button for each file right?

Reply:
yeah, we don't